google/cloud/security/inventory/pipelines/load_instances_pipeline.py

```diff
@@ -99,7 +99,22 @@ def _retrieve(self):
         return instances
 
     def run(self):
-        """Run the pipeline."""
```

Comment:
Thanks for this. Since you're the first to touch this file after we enabled doc checking can you remove the disables at the top and fix any issues as well?